docker/api/Dockerfile

```diff
@@ -34,17 +35,18 @@ COPY client.html /opt/cnaas/static
 RUN unlink /etc/nginx/sites-enabled/default
 RUN ln -s /etc/nginx/sites-available/nginx_app.conf /etc/nginx/sites-enabled/default
 RUN chown www-data:www-data /var/log/nginx
-RUN chown www-data:www-data /etc/cnaas-nms/repository.yml
+RUN chown www-data:www-data /etc/cnaas-nms/*.yml
```

Comment:
chown root:www-data?
chmod 640

Reply:
Changed to
```
RUN chown root:www-data /etc/cnaas-nms/*.yml && \
    chmod 640 /etc/cnaas-nms/*.yml
```

I get error 500 on the integration tests.

Docker log:
```
cnaas_api_1       | 2020-11-18 09:00:13,837 DEBG 'uwsgi' stdout output:
cnaas_api_1       | pcre jit disabled
cnaas_api_1       | detected number of CPU cores: 8
cnaas_api_1       | current working directory: /
cnaas_api_1       | detected binary path: /usr/local/bin/uwsgi
cnaas_api_1       | chdir() to /opt/cnaas/venv/cnaas-nms/src/
cnaas_api_1       | your memory page size is 4096 bytes
cnaas_api_1       | detected max file descriptor number: 1048576
cnaas_api_1       | - async cores set to 1000 - fd table size: 1048576
cnaas_api_1       | lock engine: pthread robust mutexes
cnaas_api_1       | thunder lock: disabled (you can enable it with --thunder-lock)
cnaas_api_1       | uwsgi socket 0 bound to UNIX address /tmp/uwsgi.sock fd 6
cnaas_api_1       | Python version: 3.7.3 (default, Jul 25 2020, 13:03:44)  [GCC 8.3.0]
cnaas_api_1       | PEP 405 virtualenv detected: /opt/cnaas/venv
cnaas_api_1       | Set PythonHome to /opt/cnaas/venv
cnaas_api_1       | Python main interpreter initialized at 0x55d5aefe7930
cnaas_api_1       | python threads support enabled
cnaas_api_1       | your server socket listen backlog is limited to 100 connections
cnaas_api_1       | your mercy for graceful operations on workers is 60 seconds
cnaas_api_1       | mapped 21036928 bytes (20543 KB) for 1000 cores
cnaas_api_1       | *** Operational MODE: async ***
cnaas_api_1       | *** uWSGI is running in multiple interpreter mode ***
cnaas_api_1       | spawned uWSGI master process (pid: 10)
cnaas_api_1       | spawned uWSGI worker 1 (pid: 11, cores: 1000)
cnaas_api_1       | spawned uWSGI mule 1 (pid: 12)
cnaas_api_1       | 
cnaas_api_1       | 2020-11-18 09:00:13,841 DEBG fd 9 closed, stopped monitoring <POutputDispatcher at 139666061079544 for <Subprocess at 139666061398384 with name nginx in state STARTING> (stderr)>
cnaas_api_1       | 2020-11-18 09:00:13,854 DEBG 'uwsgi' stdout output:
cnaas_api_1       | running "/opt/cnaas/exec-pre-app.sh" (pre app)...
cnaas_api_1       | 
cnaas_api_1       | 2020-11-18 09:00:13,859 DEBG 'uwsgi' stdout output:
cnaas_api_1       | /opt/cnaas/exec-pre-app.sh: 15: /opt/cnaas/exec-pre-app.sh: cannot create /etc/cnaas-nms/repository.yml: Permission denied
```

Reply:
Ah sorry I forgot about that, but I think it's fine if we change so www-data:www-data is owner of /etc/cnaas-nms/